src/main/java/net/dv8tion/jda/core/entities/impl/UserImpl.java

```diff
@@ -125,6 +128,12 @@ protected void handleResponse(Response response, Request request)
     }
 
     @Override
+    public List<Guild> getMutualGuilds()
```

Comment:
(This was countered in chat. Decision was not to include it here as it would be redundant to `JDA#getMutalGuilds(User...)`